src/Cafe/OS/libs/nsyshid/BackendLibusb.h

```diff
@@ -3,7 +3,7 @@
 
 #include "nsyshid.h"
 
-#if NSYSHID_ENABLE_BACKEND_LIBUSB
+#if 1
```

Comment:
If this is to use the libusb backend on Windows by default, a cleaner way would probably be setting the following to `OFF` in `CMakeLists.txt`:
https://github.com/cemu-project/Cemu/blob/64b0b85ed5fe15d17cfa6b5fce0044000b013a07/CMakeLists.txt#L103
Otherwise we'd see each USB device twice!
I honestly have no idea what a game would do in that case; from connecting two Lego Dimensions portals at the same time, my guess would be that one is just chosen at random.

Reply:
Ah this was just me because my IDE wasn't recognising the libusb backend enabled and all the code was very dim, will add this back in